packages/gatsby-plugin-feed/src/gatsby-node.js

```diff
@@ -1,12 +1,21 @@
-import fs from "fs"
+import fs from "fs-extra"
 import path from "path"
 import RSS from "rss"
 import merge from "lodash.merge"
-import mkdirp from "mkdirp"
-import { defaultOptions, runQuery, writeFile } from "./internals"
+
+import { defaultOptions, runQuery } from "./internals"
+import pluginOptionsSchema from "./plugin-options"
 
 const publicPath = `./public`
 
+const warnMessage = (error, behavior) => `
+  gatsby-plugin-feed was initialized in gatsby-config.js without a ${error}.
+  This means that we the plugin will use ${behavior}, which may not match your use case.
+  This behavior will be removed in the next major release of gatsby-plugin-feed.
+  For more info, check out: https://gatsby.app/rss-feed
+`
+
+// TODO: remove in the next major release
```

Comment:
Should we file an issue for this TODO or does that happen automatically? 🙏

Reply:
These are more for internal use--i.e. when we're ready to introduce breaking changes, we'll take that opportunity to refactor all these TODOs and clean up the code!